.github/workflows/update-smithy-version.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         id: fetch-latest
         run: |
           echo "latestSmithy=$( \
-            curl -sL https://api.github.com/repos/awslabs/smithy/releases/latest | \
+            curl -sL https://api.github.com/repos/smithy-lang/smithy/releases/latest | \
             jq -r '.tag_name')" >> $GITHUB_OUTPUT
```

Comment:
Once we switch over the CLI release, we'll start including the VERSION file in the release assets. Instead of relying on the GH api, we can use the version-file too if we wanted. Occasionally, the api is finicky.